descriptors/ModuleDescriptor-template.json

```
@@ -416,9 +416,71 @@
           "permissionsRequired": ["patron-action-session-storage.expired-session-patron-ids.collection.get"]
         }
       ]
+    },
+    {
+      "id": "check-in-operation-storage",
```

I'm not keen on `operation`, yet I've been trying to to think of an alternative, without much success. What do you think of `check-in-storage`?

```suggestion
      "id": "check-in-operation-storage",
```

---

Fine for me, do you mean to rename URL as well?

---

What I'm thinking is that the interface ID is `check-in-storage`, and the paths are something like `check-in-storage/check-ins`

---

This is assuming that FOLIO is still using plural path names for its endpoints